### PR TITLE
Adjust various log messages which had missing $expansions

### PR DESCRIPTION
**Pull request by alicetries**
Fixes various missing expansions within log messages.

### CODE REVIEW

**Comment by alicetries** [comment]
I had considered adding a null check in for link->class, but from reading conf.c it doesn't seem like should ever be NULL (as it should get replaced by the default class), but feel free to adjust if that's a bad assumption.

File: src/modules/tkl.c [hunk 1]
```diff
@@ -3706,7 +3706,7 @@ void tkl_sync_send_entry(int add, Client *sender, Client *to, TKL *tkl)
 	} else
 	{
 		unreal_log(ULOG_FATAL, "tkl", "BUG_TKL_SYNC_SEND_ENTRY", NULL,
-			   "[BUG] tkl_sync_send_entry() called for '%s' but unknown type: $tkl.type_string ($tkl_type_int)",
+			   "[BUG] tkl_sync_send_entry() called, but unknown type: $tkl.type_string ($tkl_type_int)",
```
Comment (alicetries) [inline src/modules/tkl.c:3709]: UnrealIRCd 5.2.x matches the message I've changed it to. I was a bit unclear to what the %s could expand out to (given $tkl.type_string and $tkl_type_int are already present)

File: src/log.c [hunk 2]
```diff
@@ -886,6 +886,7 @@ LogData *log_data_link_block(ConfigItem_link *link)
 	json_object_set_new(j, "hostname", json_string_unreal(link->outgoing.hostname));
 	json_object_set_new(j, "ip", json_string_unreal(link->connect_ip));
 	json_object_set_new(j, "port", json_integer(link->outgoing.port));
+	json_object_set_new(j, "class", json_string_unreal(link->class->name));
```
Comment (alicetries) [inline src/log.c:889]: I had considered adding a null check in for link->class, but from reading conf.c it doesn't seem like should ever be NULL (as it should get replaced by the default class), but feel free to adjust if that's a bad assumption.

**Comment by syzop** [comment]
Thanks! Lots of corrections, much appreciated :)

**Pull request merged**